src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/ccdcallbackhandler/CcdEvent.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@ToString
```

Comment:
I wasn't aware we're using lombok on this repo... we aren't already using it I'd prefer not to start.

Reply:
it's so much easier to write rapidly and concisely -

Reply:
I'd prefer not to use magic but we can do